.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,21 @@
+name: Check PR title
```

Comment:
instead introducing a new workflow, I would just adjust
https://github.com/shopware/shopware/blob/trunk/.danger.php#L230-L241

Reply:
it is a new workflow so check for that is very straightforward and descriptive. Lowers the barrier for newcomers to understand when needs to be improved. The `danger` script is mostly checking correctness of files, so I would leave it as a files linter. Similar workflows for the one introduces here are labelling or CLA check - related to PR process